tests/integration/panel/empty-page-tests.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
```

Comment:
I wanted to avoid having to switch to the empty page, because that would add unnecessary delay, and it would force me to make the first test asynchronous as well.